pkg/controller/plan/kubevirt.go

```diff
@@ -239,7 +239,7 @@ func (r *KubeVirt) getImporterPods(pvc *core.PersistentVolumeClaim) (pods []core
 		return
 	}
 	for _, pod := range podList.Items {
-		if strings.Contains(pod.Name, fmt.Sprintf("importer-%s-%s", r.Plan.Name, pvc.Annotations[kVM])) {
+		if strings.Contains(pod.Name, fmt.Sprintf("importer-%s", pvc.Name)) {
```

Comment:
I think there should be an annotation on the pvc `cdi.kubevirt.io/storage.import.importPodName`, but I looked in some envs and I don't see it, so perhaps it's only set on the prime PVC or something

Reply:
uhh, in L#224 we do check it:
```go
	if _, ok := pvc.Annotations[AnnImporterPodName]; !ok {
		return
	}
```
so I'm not sure what's going on

Reply:
it exists in a single importer pod. when using warm migration it won't be there, as we create importer pod per checkpoint

Reply:
so what does the if in L#224 do?

Reply:
and isn't there a label we can query instead?

Reply:
> so what does the if in L#224 do?

I think making sure our PVC is legit (part of import). We may drop it if you like.



> and isn't there a label we can query instead?

I think there isn't one.

Reply:
> > and isn't there a label we can query instead?
> 
> I think there isn't one.

ok, please check - if there is, it would be better

Reply:
>I think making sure our PVC is legit (part of import). We may drop it if you like.

you mean the annotation is set but it's an empty string?

Reply:
> > > and isn't there a label we can query instead?
> > 
> > 
> > I think there isn't one.
> 
> ok, please check - if there is, it would be better

there isn't.



> > I think making sure our PVC is legit (part of import). We may drop it if you like.
> 
> you mean the annotation is set but the value is an empty string?

once we create warm migration the VM has multiple PVCs (original and scratch), the original contains our annotation while the scratch isn't. but, looking on `getPVCs` function, we anyway get the PVC with the annotation. in my opinion we can drop this `if` at the start.